package.json

```diff
@@ -81,10 +80,12 @@
   "scripts": {
     "start": "node scripts/start.js",
     "build": "node scripts/build.js && node versioned-build.js",
-    "test": "node scripts/test.js",
+    "test": "node scripts/test.js --no-watch",
     "lint": "tslint --project tsconfig.json -c tslint.json",
     "prebuild:prod": "standard-version",
-    "build:prod": "npm run build"
+    "build:prod": "npm run build",
+    "start-server": "npm start",
+    "ci": "start-server-and-test start-server http://localhost:3000 test"
```

Comment:
Could we give this a clearer name? Like `"test:accessibility" ` so that it is clearer what the tests are testing?